elements/rh-switch/docs/20-guidelines.md

```diff
@@ -0,0 +1,253 @@
+
+<rh-cta>
+  <a href="../accessibility/#using-form-labels">Learn how to use form labels with Switch</a>
+</rh-cta>
+
+
+## Writing content
+
+In general, users scan and do not read everything, so use keywords and avoid long phrases and questions.
+
+
+### Clarity of language
+
+The status message and form label should be short and direct, not neutral or ambiguous.
+
+
+<div class="grid sm-two-columns">
+  <uxdot-best-practice do>
+    <uxdot-example slot="image" alignment="left" width-adjustment="241px">
+      <img src="{{ '../switch-language-clarity-do.svg' | url }}" alt="Magenta button, brand red default call to action, green tooltip, and dark orange switch">
+    </uxdot-example>
+    <p>Ensure the message is clear when a Switch is toggled to the On or Off position and that the form label explains the switch's purpose.</p>
+  </uxdot-best-practice>
+
+  <uxdot-best-practice dont>
+    <uxdot-example slot="image" alignment="left"  width-adjustment="254px">
+      <img src="{{ '../switch-language-clarity-do-not.svg' | url }}" alt="Magenta button, brand red default call to action, green tooltip, and dark orange switch">
+    </uxdot-example>
+    <p>Do not use a status message that does not make the switch state clear, especially if the form label is hidden.</p>
+  </uxdot-best-practice>
+</div>
+
+
+### Character count
+
+The recommended maximum character count is listed below and includes spaces.
+
+<rh-table>
+  <table>
+    <thead>
+      <tr>
+        <th scope="col" data-label="Element">Element</th>
+        <th scope="col" data-label="Character count">Character count</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td scope="col" data-label="Element">Switch status message</td>
+        <td scope="col" data-label="Character count">20</td>
+      </tr>
+      <tr>
+        <td scope="col" data-label="Element">Form label</td>
+        <td scope="col" data-label="Character count">30</td>
+      </tr>
+    </tbody>
+  </table>
+</rh-table>
+
+
+## Layout
+
+### Stacking
+
+Use a stack of Switches for situations where multiple independent options need to be turned on or off.
+
+<uxdot-example width-adjustment="193px">
+  <img src="{{ '../switch-layout-stacking.svg' | url }}" alt="four switches, half of which are on, stacked in a column">
+</uxdot-example>
+
+
+## Behavior
+
+A Switch is successfully toggled when the handle slides to the other side of the track and the status message changes. When a user toggles a Switch, the effects should start immediately without needing to save. If immediate results are not achievable, another element should be used instead (see table above in Usage section).
+
+<uxdot-example width-adjustment="319px">
+  <img src="{{ '../switch-behavior.svg' | url }}" alt="One switch that is on, next to one that is off">
+</uxdot-example>
+
+
+## Best practices
```

Comment:
The images under "Best practices" should be centered, and the example containers should be the same height. (If this is due to a difference of dimensions after exporting, I can re-export these images!)